src/main/scala/com/monsanto/arch/cloudformation/model/resource/ApplicationAutoScaling.scala

```diff
@@ -80,7 +97,7 @@ case class `AWS::ApplicationAutoScaling::ScalableTarget`(name: String,
                                                          RoleARN: Token[String],
                                                          ScalableDimension: Token[String],
```

Comment:
Should we consider adding in an enumeration for values here? Looks like `ecs:service:DesiredCount | ec2:spot-fleet-request:TargetCapacity | elasticmapreduce:instancegroup:InstanceCount | appstream:fleet:DesiredCapacity | dynamodb:table:ReadCapacityUnits | dynamodb:table:WriteCapacityUnits | dynamodb:index:ReadCapacityUnits | dynamodb:index:WriteCapacityUnits | rds:cluster:ReadReplicaCount | sagemaker:variant:DesiredInstanceCount | custom-resource:ResourceType:Property` are the only valid values. I'm guessing there is also some implicit verification that happens to make sure it matches the ServiceNamespace when the CloudFormation gets processed we may want to consider as well?

Reply:
That sounds reasonable. I don't have time to get to this right away, so if you have time to do it, that would be great.

Reply:
I tried hacking on it a bit. Adding the enum is easy enough but it might end up creating complications since it is a `Token`. I'm assuming it is a token to facilitate passing it in as a parameter? If so, I can also just make a custom parameter type for it.

Reply:
I don't know why it's a token. I don't see that keeping it a token adds any value.

Reply:
Awesome I'll remove it then and get this merged.